<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>684</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>LabAnalyser</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBarI">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadPlugin"/>
    <addaction name="actionLoad_Form"/>
    <addaction name="separator"/>
    <addaction name="actionLoadExperiment"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_Experiment"/>
    <addaction name="Close_Project"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Parameter_File"/>
    <addaction name="actionSave_Parameter_Set"/>
    <addaction name="separator"/>
    <addaction name="actionDaten_Exportieren_mat"/>
    <addaction name="actionExport_Data_h5"/>
    <addaction name="separator"/>
    <addaction name="actionMinimize_to_Tray"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionCreatePlot"/>
    <addaction name="actionCreate_Subplot"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionRemote_Connection_Port_2"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuPlot"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadExperiment"/>
   <addaction name="actionSave"/>
   <addaction name="Close_Project"/>
   <addaction name="separator"/>
   <addaction name="actionCreatePlot"/>
   <addaction name="actionCreate_Subplot"/>
   <addaction name="separator"/>
   <addaction name="actionLoadPlugin"/>
   <addaction name="actionLoad_Form"/>
   <addaction name="separator"/>
   <addaction name="actionSave_Parameter_Set"/>
   <addaction name="actionLoad_Parameter_File"/>
   <addaction name="separator"/>
   <addaction name="actionDaten_Exportieren_mat"/>
   <addaction name="actionExport_Data_h5"/>
  </widget>
  <widget class="QDockWidget" name="ParameterDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Parameter Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_10">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="TreeWidgetCustomDrop" name="ParameterTreeWidget">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::CopyAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="animated">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="StateDock">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>State Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="TreeWidgetCustomDrop" name="StateTreeWidget">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="DataDock">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Data Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="TreeWidgetCustomDrop" name="DataTreeWidget">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="animated">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="OutputDock">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Outputs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="OutputText">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionBeenden">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad_Form">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/collage-line.png</normaloff>:/icons/icons/collage-line.png</iconset>
   </property>
   <property name="text">
    <string>Load User Interface</string>
   </property>
   <property name="iconText">
    <string>Load
UI</string>
   </property>
   <property name="toolTip">
    <string>Load User Interface Files, created with QT Designer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionCreatePlot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/plot.png</normaloff>:/icons/icons/plot.png</iconset>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
   <property name="toolTip">
    <string>Create Plot Window</string>
   </property>
  </action>
  <action name="actionCreate_Subplot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/subplot.png</normaloff>:/icons/icons/subplot.png</iconset>
   </property>
   <property name="text">
    <string>Sub Plot</string>
   </property>
   <property name="toolTip">
    <string>Create SubPlot Window</string>
   </property>
  </action>
  <action name="actionLoadPlugin">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/base-station-line.png</normaloff>:/icons/icons/base-station-line.png</iconset>
   </property>
   <property name="text">
    <string>Load Device Plugin</string>
   </property>
   <property name="iconText">
    <string>Load
Device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSave_Experiment">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save-3-line.png</normaloff>:/icons/icons/save-3-line.png</iconset>
   </property>
   <property name="text">
    <string>Save Experiment As</string>
   </property>
  </action>
  <action name="actionLoadExperiment">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/folder-open-line.png</normaloff>:/icons/icons/folder-open-line.png</iconset>
   </property>
   <property name="text">
    <string>Load Experiment</string>
   </property>
   <property name="iconText">
    <string>Load
Experiment</string>
   </property>
   <property name="toolTip">
    <string>Load Experiment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Close_Project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/close-fill.png</normaloff>:/icons/icons/close-fill.png</iconset>
   </property>
   <property name="text">
    <string>Close Experiment</string>
   </property>
   <property name="iconText">
    <string>Close
Experiment</string>
   </property>
   <property name="toolTip">
    <string>Close Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionDaten_Exportieren_mat">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/file-code-line (1).png</normaloff>:/icons/icons/file-code-line (1).png</iconset>
   </property>
   <property name="text">
    <string>Export Data (*.mat)</string>
   </property>
   <property name="iconText">
    <string>Export
(*.mat)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save-3-fill.png</normaloff>:/icons/icons/save-3-fill.png</iconset>
   </property>
   <property name="text">
    <string>Save Experiment</string>
   </property>
   <property name="iconText">
    <string>Save
Experiment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMinimize_to_Tray">
   <property name="text">
    <string>Minimize to Tray</string>
   </property>
  </action>
  <action name="actionAbout_LabAnalyzer">
   <property name="text">
    <string>About LabAnalyser</string>
   </property>
  </action>
  <action name="actionLoad_Parameter_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/list-settings-line.png</normaloff>:/icons/icons/list-settings-line.png</iconset>
   </property>
   <property name="text">
    <string>Load Parameter File</string>
   </property>
   <property name="iconText">
    <string>Load
Parameter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionSave_Parameter_Set">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/list-settings-fill.png</normaloff>:/icons/icons/list-settings-fill.png</iconset>
   </property>
   <property name="text">
    <string>Save Parameter Set</string>
   </property>
   <property name="iconText">
    <string>Save
Parameter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/sym_about.png</normaloff>:/icons/icons/sym_about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExport_Data_h5">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/file-code-line (1).png</normaloff>:/icons/icons/file-code-line (1).png</iconset>
   </property>
   <property name="text">
    <string>Export Data (*.h5)</string>
   </property>
   <property name="iconText">
    <string>Export 
(*.h5)</string>
   </property>
  </action>
  <action name="actionRemote_Connection_Port">
   <property name="text">
    <string>Remote Connection Port</string>
   </property>
  </action>
  <action name="actionRemote_Connection_Port_2">
   <property name="text">
    <string>Remote Connection Port</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TreeWidgetCustomDrop</class>
   <extends>QTreeWidget</extends>
   <header>treewidgetcustomdrop.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
